Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -146,6 +149,29 @@ extension Event.HumanReadableOutputRecorder {
 
     return (errorIssueCount, warningIssueCount, knownIssueCount, totalIssueCount,  description)
   }
+  
+  /// Returns a formatted string describing the number of arguments in a test,
+  /// based on verbosity level.
+  ///
+  /// - Parameters:
+  ///   - isParameterized: To check test cases count should be included.
+  ///   - count: The number of test cases in the test.
+  ///   - verbosity: The verbosity level. If it's very verbose or higher,
+  ///     the test cases count might get included.
+  ///
+  ///
+  /// - Returns: A string describing the number of test cases in the test,
+  ///   or an empty string if it's not very verbose level.
+  ///
+  private func _includeNumberOfTestCasesIfNeeded(
```

Comment:
You probably don't need this helper function and can just construct the string in place in the `.testEnded` handler below.